### ISSUE TITLE
* operator within parens and no space issues

### ISSUE THREAD

**Issue by bgriffith** [tonyganch/gonzales-pe#107]
First off this is in scss and appears to happen in both the current and latest dev branch.

If I wrap calculations that use the `*` operator in parenthesis and don't have any spaces, the node structure bundles the operator within an ident of the following number instead of an operator.

The following are ok:

``` scss
// These are all ok
$foo: (1+1);
$bar: (2-1);
$baz: (1/2);

// The * operator works ok without parens - see below

$foo: 2*1;
```

<img width="243" alt="screen shot 2015-10-12 at 18 01 41" src="https://cloud.githubusercontent.com/assets/2854338/10433635/72dd11c2-710b-11e5-84c7-570a39d07470.png">

But when I wrap in parenthesis, the node structure loses the operator node

``` scss
// This causes the issue with nodes below
$qux: (2*1);
```

<img width="303" alt="screen shot 2015-10-12 at 17 55 07" src="https://cloud.githubusercontent.com/assets/2854338/10433550/bb64910a-710a-11e5-81ed-ba0b59abfc2a.png">

**Comment by tonyganch** [comment]
Those two lines should be removed from all syntaxes: https://github.com/tonyganch/gonzales-pe/blob/dev/src/css/parse.js#L1152

@bgriffith, you can make the change yourself or wait till weekend, I'll have time for the fix then.

**Comment by tonyganch** [comment]
Should be fixed in `@3.0.2` for both sass and scss. Please let me know if the issue is still there.
